client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import EducationalContent from '../educational-content';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+/**
+ * Image dependencies
+ */
+import earnCardPrompt from 'assets/images/customer-home/illustration--business.svg';
+
+const EducationEarn = ( { siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<EducationalContent
+			title={ translate( 'Make money from your website' ) }
+			description={ translate(
+				'Accept credit and debit card payments on your website for just about anything.'
+			) }
+			links={ [
+				{
+					calypsoLink: true,
```

Comment:
Maybe `isRelative` or let the component sniff that out.

Reply:
Alternatively we can mark isInlineSupportLink and isExternal with the default being relative.